Doc/whatsnew/3.9.rst

```diff
@@ -105,6 +105,16 @@ Merge (``|``) and update (``|=``) operators have been added to the built-in
 :class:`dict` class.  See :pep:`584` for a full description.
 (Contributed by Brandt Bucher in :issue:`36144`.)
 
+PEP 616: New removeprefix() and removesuffix() string methods
+-------------------------------------------------------------
+
+:meth:`str.removeprefix(prefix)<str.removeprefix>` and
+:meth:`str.removesuffix(suffix)<str.removesuffix>` have been added
+to easily remove an unneeded prefix or a suffix from a string. Corresponding
+``bytes``, ``bytearray``, and ``collections.UserString`` methods have also been
+added. See :pep:`616` for a full description. (Contributed by Dennis Sweeney in
+:issue:`18939`.)
```

Comment:
This is not [18939](https://bugs.python.org/issue18939), which is about "Venv docs regarding original python install". Shouldn't this be [39939](https://bugs.python.org/issue39939)?

Reply:
Right, it's a typo error: @sweeneyde: can you please propose a PR to fix the typo?

Reply:
Or @elazarg: Do you want to propose a PR to fix the typo?

Reply:
Sure. I thought it might be overkill :)

Reply:
Oops -- It looks like that's the GitHub PR number rather than the bpo number. I can't make a PR tonight so feel free to change it. If not, I can fix it tomorrow.

Reply:
Done.